80-insights-register.preset

```diff
@@ -2,3 +2,5 @@ enable insights-register.path
 enable insights-unregister.path
 enable rhcd.path
 enable rhcd-stop.path
+enable insights-unregistered.path
+enable rhcd.service
```

Comment:
Oh, I think it will be better to split the preset file in two:
- `80-insights-register.preset`
- `80-rhcd-register.preset`

As it, we won't have issue on `RHEL8/9` but we will have on `RHEL7.9`, as the `rhcd` `systemd` bits won't be existing on that release.

Reply:
Added and updated the PR info.